packages/babel-core/test/helpers/esm.js

```diff
@@ -56,7 +56,16 @@ async function spawn(runner, filename, cwd = process.cwd()) {
     process.execPath,
     // pass `cwd` as params as `process.cwd()` will normalize `cwd` on macOS
     [require.resolve(`../fixtures/babel-${runner}.mjs`), filename, cwd],
-    { cwd, env: process.env },
+    {
+      cwd,
+      env: {
+        ...process.env,
+        NODE_OPTIONS:
+          parseInt(process.versions.node) >= 23
+            ? "--disable-warning=ExperimentalWarning"
```

Comment:
Does node 23.1 updated the error message? We already removed the experimental warning from stderr.

Plus if we provide other NODE_OPTIONS, this will replaced the provided options and with `--disable-warning` only.

Reply:
You can see the failure at https://github.com/babel/babel/actions/runs/11509402711/job/32039492498

Reply:
Could you open another PR? 🙏